src/store/user-store.js

```diff
@@ -3,6 +3,8 @@ import { makeAutoObservable } from "mobx";
 class UserStore {
   isLoggedIn = false;
 
+  userResponse = {};
```

Comment:
Step 2: We need to then create a variable in the user store to hold the user data from spotify.

Once we make a fetch call to `https://api.spotify.com/v1/me`, the intention here is to store that data to `userResponse`. Previously, you named this `userData` and I kinda like that more so feel free to keep it that way if you want. If you prefer to name the variable `userResponse`, that's fine too. Either way it works!